_rest_routes.inc.php

```diff
@@ -150,6 +151,14 @@
         $data = (array) (json_decode(file_get_contents("php://input")));
         return (new PractitionerRestController())->patch($prid, $data);
     },
+    "GET /api/allergy_intolerance" => function () {
+        RestConfig::authorization_check("patients", "med");
+        return (new AllergyIntoleranceRestController())->getAll($_GET);
+    },
+    "GET /api/allergy_intolerance/:id" => function ($id) {
+        RestConfig::authorization_check("patients", "med");
+        return (new AllergyIntoleranceRestController())->getOne($id);
+    },
```

Comment:
@yash, I just realized we already have allergy api endpoints. Shouldn't we be using the allergy api endpoints that already exist?

Reply:
@yashrajbothra , I messed up on the @ above; check out my post above